subprocess.hpp

```diff
@@ -1080,12 +1080,12 @@ inline void Popen::start_process() throw (CalledProcessError, OSError)
   execute_process();
 }
 
-inline int Popen::wait() throw (OSError)
+inline int Popen::wait() noexcept
 {
   int ret, status;
   std::tie(ret, status) = util::wait_for_child_exit(pid());
   if (ret == -1) {
-    if (errno != ECHILD) throw OSError("waitpid failed", errno);
+    if (errno != ECHILD) OSError("waitpid failed", errno);
```

Comment:
What does this do ? It just creates a temporary OSError object ? I wanted to raise an exception here.

Reply:
Yep.  It does nothing but other than creating a `std:runtime_error` to be exact.  I have an update for this to bring back the `throw`.